src/Traits/ComparisonTrait.php

```diff
@@ -240,6 +240,66 @@ public function isTomorrow()
     }
 
     /**
+     * Determines if the instance is within the next week
+     *
+     * @return bool
+     */
+    public function isNextWeek()
+    {
+        return $this->toDateString() === static::now($this->tz)->addWeek()->toDateString();
```

Comment:
@lorenzo Thank you for your review!

> this method seems incomplete

Sorry. I re-checked them and fixed up. It should now work as described.

And I have a question about this.

```php
static::now($this->tz) // Change to
$this->copy();
```

https://github.com/briannesbitt/Carbon/pull/846
I think that every method is implemented to make a judgment based on "now" in this PR.

If we use `$this->copy()`, the behavior won't change?